### PR TITLE
101 est refinement

### PR DESCRIPTION
**Pull request by matt-auckland**

### CODE REVIEW

File: src/app/components/status/status.component.ts [hunk 1]
```diff
@@ -53,22 +54,47 @@ export class StatusComponent implements OnInit {
...
-    this.changellyFeeTwo= formData.changellyFeeTwo
-    this.etaMin = this.genFuncs.calculateOrderEst(formData.eta[0]).toLocaleString()
-    this.etaMax = this.genFuncs.calculateOrderEst(formData.eta[1]).toLocaleString()
+    this.estConvFromNav = this.formatNumber(formData.estConvFromNav)
+    this.estimatedFees = this.formatNumber(formData.estimatedFees)
+    this.etaMin = this.genFuncs.calculateOrderEst(formData.estTime[0]).toLocaleString()
+    this.etaMax = this.genFuncs.calculateOrderEst(formData.estTime[1]).toLocaleString()
+  }
+
+  formatNumber(numberStr) {
```
Comment (craigmacgregor) [inline src/app/components/status/status.component.ts]: is this all necessary? surely there's a number formatter built into angular?

Comment (craigmacgregor) [reply src/app/components/status/status.component.ts]: {{ numberStr | number }} in the view i think

**Comment by craigmacgregor** [comment]
No matter what combinations i put in, i get:

Est. Arrival: 12 mins ~ 70 mins

Doesn't seem right.

**Comment by matt-auckland** [comment]
> No matter what combinations i put in, i get:
> Est. Arrival: 12 mins ~ 70 mins
> Doesn't seem right.
This has been fixed

> Also, NavTech processing time should be 5 - 10 minutes. But we can always update these times later.

This is handled in the `./server/config.js` file by the timeConsts obj. I've updated it to 5,10 (min,max) on my side but you'll need to update it on your side too
```  timeConsts: {
    changelly: [5,30],
    navTech: [5,10],
  },```

**Pull request merged**